examples/custom-routes-proxying/app/hello/[slug]/page.tsx

```diff
@@ -0,0 +1,10 @@
+import Link from "next/link";
+
+export default function About({ params }) {
```

Comment:
Let's make sure we run Prettier.

We'll also want to type `params` here.

Reply:
@leerob I have made the changes.